### PR TITLE
Game instructions

### PR DESCRIPTION
**Pull request by DcRonan**
### README.md

1. Displayed gif of the game
2. Have given instructions on how to play
3. Explained how to install and download on the local machine.
5. Created a live demo link.

**FYI** The live demo is for the 'development' branch and will change when merged.

### CODE REVIEW

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Dan Ronan & Bello Babakolo** 
Great job following the improvements of the last review :muscle:  :muscle: 
Your project meets most of the requirements :clap: :clap: :clap:  
We still need to fix some details:

- [ ] In the "Steps" section of your README file you have a typo, change 'wil' with 'will'
- [ ] In the "install" section of your README file, the user needs to run "bundle install" before running the project

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

File: README.md [hunk 1]
```diff
@@ -26,20 +46,25 @@ Additional description about the project and its features...
...
 
 ### Install
+**To get this working on your local computer, follow the steps below!**
+
 ```
-Download code
+1. Install Ruby
+2. Open the Terminal
+3. Run 'bundle install'
+4. Download code by entering 'git clone git@github.com:DcRonan/Tic-Tac-Toe.git' in the terminal
```
Comment (maelfosso) [inline README.md]: Don't you think that that 4-th step comes before the 3-rd step? 
I mean we download the code, `cd` into the folder and then run `bundle install`.

File: README.md [hunk 2]
```diff
@@ -26,20 +46,25 @@ Additional description about the project and its features...
...
 ```
 ### Run
 ```
 1. Open text editor
```
Comment (maelfosso) [inline README.md]: Why should we open the text editor to run the code?

File: README.md [hunk 3]
```diff
@@ -26,20 +46,25 @@ Additional description about the project and its features...
...
 2. Open command prompt
-3. Enter directory of 'main.rb'
-4. Type 'ruby main.rb'
+3. Enter directory of 'main'
```
Comment (maelfosso) [inline README.md]: It's preferable to say, `cd bin/`

File: README.md (hunk 1 above)
Comment (maelfosso) [reply README.md]: Also, it's preferable to use the `http` download method, I mean 
`https://github.com/DcRonan/Tic-Tac-Toe.git`

Suggest the **ssh** method to people allowed to change the code

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Changes required :red_circle: 

Hello,

Thank you for applying the suggested changes.
There are some small things to change in your README
Kindly read the comments left below, make the suggested changes and submit a new code review request

Thanks

Mael FOSSO

File: README.md [hunk 4]
```diff
@@ -26,20 +46,25 @@ Additional description about the project and its features...
...
 ```
 ### Run
 ```
-1. Open text editor
-2. Open command prompt
-3. Enter directory of 'main.rb'
-4. Type 'ruby main.rb'
+1. Open terminal
+2. Run 'cd Tic-Tac-Toe' to enter main directory
+3. Run 'cd bin/main' to start the game
```
Comment (maelfosso) [inline README.md]: To start the game is `ruby bin/main`

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: 

Thank you for applying the suggested changes
You did a really good job
**PS**: You can merge it but before read the comment left below.

Good continuation :handshake: 

Mael FOSSO

**Pull request merged**